utils/Newton_Method.py

```diff
@@ -0,0 +1,39 @@
+from sympy import symbols, diff, sympify
+
+# Input function, value of x0, symbol, and epsilon
+f_x_input = input("Enter the function (must be quadratic or greater degree, e.g., x**2 + 3*x + 2): ")
+y = float(input("Enter the value of x0: "))
```

Comment:
Please make the prompt messages more informative.
Instead of asking the value of X0, ask what does X0 mean.
You can provide symbols at the end of the prompt message
Example:
Enter the value of initial approximation (X0): 
Do the same for all the inputs